src/main/scala/mesosphere/marathon/core/matcher/reconcile/impl/OfferMatcherReconciler.scala

```diff
@@ -67,6 +67,9 @@ private[reconcile] class OfferMatcherReconciler(taskTracker: TaskTracker, groupR
                   resources = spuriousResources.to[Seq]
                 )
               TaskOpWithSource(source(offer.getId), unreserveAndDestroy)
+            //          case (taskId, spuriousResources) if shouldRefreshLastSeen(taskId) =>
+            //            // this is slightly silly but we need to refresh
```

Comment:
as far as I can see, `shouldRefreshLastSeen` is not implemented – is that WIP or should it be kept for future reference?